app/org/sagebionetworks/bridge/upload/UploadValidationTaskFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.sagebionetworks.bridge.upload;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.dao.UploadDao;
+import org.sagebionetworks.bridge.models.studies.Study;
+import org.sagebionetworks.bridge.models.upload.Upload;
+
+@Component
+public class UploadValidationTaskFactory {
```

Comment:
RE: your comment: we are welded to Spring at this point. If it's less code to use context.getBean(), please do it. That's what Spring does after all, it makes objects.